readme.md

```diff
@@ -145,11 +145,16 @@ Handle loading start.
 
 #### `PillOptions.onMounting()`
 ```
-(page:Page, url:URL) -> void
+(page:Page, url:URL, element:HTMLElement) -> void
 ```
-
 Fires everytime new content is about to be loaded to the DOM.
 
+#### `PillOptions.onUnmounting()`
```

Comment:
Could you move this block after onReady subsection?

Subsections in Pill documentation should go in lexicographical order, not in logical.

Reply:
I've moved this block and I have 2 other PRs out there